pytorch_lightning/loops/processors/iterator_batch_processor.py

```diff
@@ -158,16 +161,18 @@ def _build_kwargs(self, dataloader_iter: Iterator, batch_idx: int) -> Dict[str,
         """Builds the keyword arguments for training_step
 
         Args:
-            dataloader_iter: The dataloader to pass
+            batch: the batch to train on
```

Comment:
This doesn't seem to be correct. It still gets the dataloader_iter (at least in the call above)

Reply:
Done !